src/modules/shared/pages/innovation/support/support-summary-list.component.ts

```diff
@@ -96,6 +98,18 @@ export class PageInnovationSupportSummaryListComponent extends CoreComponent imp
       subscriptions.organisationsList = this.organisationsService.getOrganisationsList({ unitsInformation: true });
     }
 
+    const innovationAssessmentsObservable: ObservableInput<InnovationAssessmentListDTO[]> =
+      this.innovationsService.getInnovationAssessmentsList(this.innovation.id);
+
+    innovationAssessmentsObservable.subscribe({
+      next: (data: InnovationAssessmentListDTO[]) => {
+        this.innovationAssessmentsList = data;
+      },
+      error: error => {
+        console.error('Error fetching innovation assessments list', error);
+      }
+    });
+
     forkJoin(subscriptions).subscribe({
       next: results => {
```

Comment:
Here you should add `innovationAssessmentsObservable: ObservableInput<InnovationAssessmentListDTO[]>`  into ` const subscriptions: { ... }`  and retrieve the result inside the `forkJoin`